contracts/modules/TransferManager/GeneralTransferManager.sol

```diff
@@ -200,12 +187,39 @@ contract GeneralTransferManager is ITransferManager {
         public
         withPerm(WHITELIST)
     {
-        //Passing a _time == 0 into this function, is equivalent to removing the _investor from the whitelist
-        whitelist[_investor] = TimeRestriction(_fromTime, _toTime, _expiryTime, _canBuyFromSTO);
+        uint8 canBuyFromSTO = 0;
+        if (_canBuyFromSTO) {
+            canBuyFromSTO = 1;
+        }
+        _modifyWhitelist(_investor, uint64(_fromTime), uint64(_toTime), uint64(_expiryTime), canBuyFromSTO);
         /*solium-disable-next-line security/no-block-members*/
         emit ModifyWhitelist(_investor, now, msg.sender, _fromTime, _toTime, _expiryTime, _canBuyFromSTO);
```

Comment:
everything can be done(canBuyFromSTO check , event emission)  into  `_modifyWhitelist()` so you don't need to write the same thing in the `modifyWhitelistMulti()` again

Reply:
Done